node/lib/ITranslator.ts

```diff
@@ -0,0 +1,29 @@
+
+/**
+ * Interface that all translators implement.
+ */
+export interface ITranslator {
+    /**
+     * Creates an instance of the device, given device properties obtained from onboarding.
+     * This method is optionally async, meaning it may return either an immediate device
+     * instance or a promise for a device instance.
+     */
+    createDevice(deviceProps: any): IDevice | Promise<IDevice>;
```

Comment:
>createDevice(deviceProps: any): IDevice | Promise<IDevice [](start = 4, length = 57)

how ITranslator design addresses the fact that adding to a (AllJoyn) bridge is optional ?